client/src/components/Questions/A-Components/Answer.jsx

```diff
@@ -9,6 +9,18 @@ const Answer = (props) => {
   return (
     <div className="answer-container">
       <h1><strong>A: </strong>{props.body}</h1>
+      <div className="thumbnail-container">
+        {props.photos.map((photo, idx) => {
+          return (
+            <img
+              key={idx}
+              className='thumbnail'
+              src={photo.url}
+              alt={''}
+            />
+          );
+        })}
+      </div>
       <span className="a-details">
         {props.name === 'Seller' ?
```

Comment:
nit: ids should be unique on the document, I think this id="a.name" should be a class instead
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id